ui/pages/confirmation/templates/index.js

```diff
@@ -1,5 +1,9 @@
 import { omit, pick } from 'lodash';
 import { ApprovalType } from '@metamask/controller-utils';
+import {
+  APPROVAL_TYPE_RESULT_ERROR,
```

Comment:
These constants were just to avoid the dependency on `controller-utils` in the `ApprovalController`.

Is it best if we use the `ResultSuccess` and `ReesultError` values from the `ApprovalType` enum?

Reply:
Patch removed and bumped `@metamask/approval-controller` and  `@metamask/controller-utils`